business/workloads.go

```diff
@@ -440,6 +445,38 @@ func (in *WorkloadService) GetPodLogs(namespace, name string, opts *LogOptions)
 	return in.getParsedLogs(namespace, name, opts)
 }
 
+func isAccessLogEmpty(al *parser.AccessLog) bool {
+	if al == nil {
```

Comment:
I think the Timestamp is always needed to be successful,  perhaps we could move that one test up here and make this more efficient?

`if al == nil || al.Timestamp == ""`

Reply:
Sure I'll move that up